articles/templates/articles/blocks/chapter.html

```diff
@@ -9,7 +9,7 @@ <h2 class="chapter-heading narrow-content">{{ self.heading }}</h2>
          <div class="container-clear-none">
              <section class="block-Embed narrow-content">
                <div class="embed-responsive embed-responsive-16by9">
-                 {% include_block block.value.url|embed:720 %}
+                 {{ block.value.url|embed:720 }}
                </div>
```

Comment:
I don't think include_block was need here since we are grabbing the value of block rather than rendering it directly? Or did you add this to fix something?